### PR TITLE
Replaced "malt" in routes

### PR DESCRIPTION
**Pull request by eyesurewould**
#24 Replaced "malt" with "fermentable" in every applicable route in the front end.

Note: Did NOT change routes of service calls, just for routes that would impact the address bar (e.g. visible to a user).

### CODE REVIEW

**Comment by eyesurewould** [comment]
Missed a route...

**Comment by buildabeer** [comment]
Is that 401 error still happening?

On the recipes page, for the fermentables tab, can you change it to 'Fermentables' in the header and 'Fermentables Used' underneath of that.

I think it's good after that!

**Comment by eyesurewould** [comment]
> Is that 401 error still happening?

Yes.



> On the recipes page, for the fermentables tab, can you change it to 'Fermentables' in the header and 'Fermentables Used' underneath of that.

Done.

File: brew_wizard_ui/src/app/malt/malt.service.ts [hunk 1]
```diff
@@ -14,27 +14,27 @@ export class MaltService {
   constructor(private http: HttpClient) { }
 
   getMalts(): Observable<IMalt[]> {
-    return this.http.get(`${environment.token_auth_config.apiBase}/malts`)
```
Comment (buildabeer) [inline brew_wizard_ui/src/app/malt/malt.service.ts]: This file is calls to the backend which have not been changed to fermentables, right?

File: brew_wizard_ui/src/app/yeast/yeast-relations/yeast-relations.component.ts [hunk 2]
```diff
@@ -40,7 +40,7 @@ export class YeastRelationsComponent implements OnInit {
         },
         error => {
           this.name_complete = true;
-          this.errorMessage = 'Problem with the service. Please try against later.';
```
Comment (buildabeer) [inline brew_wizard_ui/src/app/yeast/yeast-relations/yeast-relations.component.ts]: Ya make one typo and the damn thing gets propogated everywhere.

**Comment by eyesurewould** [comment]
I undid that route change in a subsequent PR (meaning I put it back to
"malt"). Let me know if you don't see it.

**Pull request merged**